clients/js/src/hooked/AssociatedToken.ts

```diff
@@ -8,14 +9,15 @@ export function findAssociatedTokenPda(
     mint: PublicKey;
     /** The owner of the token account */
     owner: PublicKey;
+    /** The Token or Token2022 Program id */
```

Comment:
Yes, I like your idea of using strings for token program name. Let's do that!

Reply:
https://github.com/metaplex-foundation/mpl-toolbox/pull/12/commits/e8306418ea63a8dd4003419e8691a83719f95eb9